Makefile

```diff
@@ -47,3 +47,8 @@ docker-push:
 
 docker-tag:
 	docker tag $(IMAGE):$$(git rev-parse HEAD) $(IMAGE):latest
+
+# npm install -g prettier
+manifests:
+	controller-gen all
```

Comment:
We've not got any instructions on how to install this :( can we pin this using dep and invoke `go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all` instead? That way we know everyone runs a consistent version.

Reply:
dep is refusing to install this dependency as it "doesn't contain usable Go code". I'm going to move this issue to an Issue to unblock this PR.

Reply:
https://github.com/gocardless/theatre/issues/17